addons/sys_antenna/CfgAcreComponents.hpp

```diff
@@ -102,5 +113,24 @@ class CfgAcreComponents {
         connector = ACRE_CONNECTOR_3_8;
         height = 3.39;
         binaryGainFile = QPATHTOF(binary\AS-1729_VRC_gain.aba);
+        compatibleRadios[] = {"ACRE_VRC64"};
+    };
+
+    class ACRE_243CM_VHF_TNC: ACRE_BaseAntenna {
+        name = "2.43 Meter VHF Ground Spike Antenna";
```

Comment:
Stringtable all those antennas?

Reply:
They are not even shown. It's only internal